### DISCUSSION TITLE
Can I use FLAML for hyperparameters optimization of my own model?

### DISCUSSION THREAD

**Discussion by gregheymans** [microsoft/FLAML#83] [Q&A]
Hi all,
I have my own model with its own hyperparameters. I was wondering if I can use FLAML for hyperparameters optimization only for my own model.
Best,
Greg

**Comment by sonichi** [comment]
Hi Greg, does your model have scikit-learn style API like fit() and predict()? If so, you can easily add it like in this example: https://github.com/microsoft/FLAML/blob/main/notebook/flaml_automl.ipynb (Section 3. Customized Learner)

And specify `estimator_list=['your_learner']` when running `AutoML.fit()`.
